### PR TITLE
EZ-621: Added fix for the cache issue while accessing decoupled preview of unpublished nodes using the access token.

### PR DESCRIPTION
**Pull request by Vinay15**

### CODE REVIEW

File: src/EventSubscriber/EzcontentJsonapiSubscriber.php [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+   */
+  protected $configFactory;
+
+  /**
+   * Constructs a EzcontentJsonapiSubscriber object.
+   *
+   * @param \Drupal\Core\Config\ConfigFactoryInterface $config_factory
+   *   The config factory.
+   */
+  public function __construct(ConfigFactoryInterface $config_factory) {
```
Comment (arshadkhan53) [inline src/EventSubscriber/EzcontentJsonapiSubscriber.php:30]: We can use camelCase for all variable name, rest looks good.

**Review by arshadkhan53 — CHANGES_REQUESTED** [review]

File: src/EventSubscriber/EzcontentJsonapiSubscriber.php (hunk 1 above)
Comment (Vinay15) [reply src/EventSubscriber/EzcontentJsonapiSubscriber.php:30]: Hey @arshadkhan53 Not sure where and why we need camelCase, this is in sync with core format. Please share an example.

**Review by vaibhavjain-in — APPROVED** [review]
LGTM

**Review by arshadkhan53 — APPROVED** [review]
LGTM

**Pull request merged**